### PR TITLE
Fix broken link for "Weave Net" documentation in "Installing Addons" page

### PR DESCRIPTION
**Pull request by dev-johnn**
Updated the link to "Weave Net" documentation to https://github.com/weaveworks/weave  as suggested in https://github.com/kubernetes/website/issues/45201#issue-2141231774

Closes https://github.com/kubernetes/website/issues/45201

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle Fix broken link for "Weave Net" documentation in "Installing Addons" page
_(Fixing the typo in title)_

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ installation instructions. The list does not try to be exhaustive.
 * [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA
   networking solution for Kubernetes. Spiderpool is supported on bare metal, virtual machines,
   and public cloud environments.
-* [Weave Net](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/)
+* [Weave Net](https://github.com/weaveworks/weave )
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/cluster-administration/addons.md:85]: Minor nit: There seems to be an extra space at the end of the link which could be easily removed.
```suggestion
* [Weave Net](https://github.com/weaveworks/weave)
```

**Review by Arhell — COMMENTED** [review]
lgtm

**Review by vaibhav2107 — APPROVED** [review]
/lgtm

**Comment by reylejano** [comment]
Hmm, the updated link proposed in this PR links to a repo with the last commit is from 2 years ago
Weaveworks is shutting down, I propose to remove the line

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ installation instructions. The list does not try to be exhaustive.
 * [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA
   networking solution for Kubernetes. Spiderpool is supported on bare metal, virtual machines,
   and public cloud environments.
-* [Weave Net](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/)
+* [Weave Net](https://github.com/weaveworks/weave)
```
Comment (reylejano) [inline content/en/docs/concepts/cluster-administration/addons.md:85]:
```suggestion
```
Since Weave Works is shutting down, I propose to remove the line

Comment (kingdonb) [reply content/en/docs/concepts/cluster-administration/addons.md:85]: I have created:

*  https://github.com/kubernetes/website/pull/45445

Hopefully we can keep Weave Net in the guide 👍

**Pull request closed without merging**